app/src/main/java/com/example/sun/data/repository/source/ remote/CurrentWeatherRemoteDataSource.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.sun.data.repository.source.remote
+
+import com.example.sun.data.model.CurrentWeather
+import com.example.sun.data.repository.source.CurrentWeatherDataSource
+
+class CurrentWeatherRemoteDataSource : CurrentWeatherDataSource.Remote {
+    override fun getCurrentWeather(listener: OnResultListener<MutableList<CurrentWeather>>) {
```

Comment:
Sử dụng List thay vì Mutable List vì data trả về từ Data Source CỐ ĐỊNH. Việc xử lý tiếp theo cần tới mutable thì nó ở Repository
```suggestion
    override fun getCurrentWeather(listener: OnResultListener<List<CurrentWeather>>) {
```